lib/cancan/ability.rb

```diff
@@ -340,5 +354,25 @@ def default_alias_actions
         :update => [:edit],
       }
     end
+
+    def list_abilities
```

Comment:
I'd be :+1: in making this public and making naming it to something like `permissions` or something a little more indicative that it provides a hash of permissions rather than a list of `Ability`s to avoid confusion.